packages/instantsearch.css/CHANGELOG.md

```diff
@@ -3,6 +3,22 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [8.1.0](https://github.com/algolia/instantsearch/compare/instantsearch.css@8.0.0...instantsearch.css@8.1.0) (2023-09-19)
+
+
+### Bug Fixes
+
+* **prettier:** consistent version ([#5850](https://github.com/algolia/instantsearch/issues/5850)) ([ca59c6d](https://github.com/algolia/instantsearch/commit/ca59c6dbd5c9eac4e2e0179a24e39bca997ae141))
+
+
+### Features
+
+* rename packages ([#5787](https://github.com/algolia/instantsearch/issues/5787)) ([c133170](https://github.com/algolia/instantsearch/commit/c133170e563592dfc15a95daced1f8447327a09a))
```

Comment:
I'm thinking this is incorrectly affected to this package because we have whitespace diffs in the Readme. Should we leave it? I'm thinking if we cancel the feat and make it a patch it will come back next time. Wdyt?